lib/papelillo/mocks/http_mock.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Papelillo.Mocks.HttpMock do
```

Comment:
I think this should be called Stubs, so it would be something like,

```suggestion
defmodule Papelillo.Stubs.HttpStub do
```

Reply:
oh now I see, sorry for the mess